services/bots/src/github-webhook/handlers/review_drafter.ts

```diff
@@ -0,0 +1,81 @@
+import {
+  PullRequestReadyForReviewEvent,
+  PullRequestReviewSubmittedEvent,
+} from '@octokit/webhooks-types';
+import { EventType, HomeAssistantRepository } from '../github-webhook.const';
+import { WebhookContext } from '../github-webhook.model';
+import { BaseWebhookHandler } from './base';
+
+const MESSAGE_ID = '<!-- ReviewDrafterComment -->';
+
+const REVIEW_COMMENT = `${MESSAGE_ID}
+Please take a look at the requested changes, and use the **Ready for review** button when you are done, thanks :+1:
+For more information about this flow [take a look at this blog post](replace_with_blog_post_when_it_exist).
```

Comment:
Should this be on a new line? 
```suggestion

For more information about this flow [take a look at this blog post](replace_with_blog_post_when_it_exist).
```

Reply:
With URL

Should this be on a new line?  Also let's make it cursive.

```suggestion

[_Learn more about our pull request process._](https://developers.home-assistant.io/blog/2023/02/07/introducing-PR-drafting-in-reviews)
```